.gitignore

```diff
@@ -61,3 +61,5 @@ typings/
 # next.js build output
 .next
 
+__snapshots__/
```

Comment:
You should commit snapshots, otherwise what's the profit of having them?

Reply:
corrected